homeassistant/components/insteon/utils.py

```diff
@@ -268,6 +278,13 @@ async def async_srv_scene_off(service):
     hass.services.async_register(
         DOMAIN, SRV_SCENE_OFF, async_srv_scene_off, schema=TRIGGER_SCENE_SCHEMA
     )
+
+    hass.services.async_register(
```

Comment:
We prefer to use our entity service helper to register services that target entity methods.
https://developers.home-assistant.io/docs/dev_101_services#entity-services